XenAdmin/Wizards/NewVMWizard/Page_InstallationMedia.cs

```diff
@@ -64,6 +65,8 @@ public Page_InstallationMedia()
 
         public bool ShowBootParameters { get { return !SelectedTemplate.IsHVM(); } }

```

Comment:
On the NewVMWizard we have this line:
`page_3_InstallationMedia.DisableStep = noInstallMedia && !page_3_InstallationMedia.ShowBootParameters;`
Say noInstallMEdia is false; if ShowBootParameters is false, then ShowBootParameters is true, but the user won't see it because the page will be disabled. I think we can remove the line.

I also think it adds complexity if we have two properties that are checking the same thing. I suggest removing ShowBootParameters and using only ShowBootModesControl, and also renaming it to IsSelectedTemplateHVM so it's more generic and not control specific.